abs.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
```

Comment:
please remove this channel information from abs.yaml

Reply:
Can I wait till upload to do this since I'm working on a dependency chain?